app/services/open_investment_time_calculator.rb

```diff
@@ -0,0 +1,68 @@
+class OpenInvestmentTimeCalculator
+  def initialize(time_entries)
+    @newest_time_entries = time_entries
+    @aggregated_times_per_employee = nil
+
+    @employee_credentials = nil
+    @employee_id = nil
+    @employee_time_entries = nil
+    @employee = nil
+    @employee_investment_time = nil
```

Comment:
Could this be combined into a hash? Or an [*OpenStruct*](https://ruby-doc.org/stdlib-2.3.1/libdoc/ostruct/rdoc/OpenStruct.html)? Or would it be nice to have an own class to work with?